packages/grafana-ui/src/components/Combobox/MultiCombobox.tsx

```diff
@@ -354,29 +365,46 @@ export const MultiCombobox = <T extends string | number>(props: MultiComboboxPro
                       key={`${item.value}-${index}`}
                       data-index={index}
                       {...itemProps}
-                      className={cx(styles.option, { [styles.optionFocused]: highlightedIndex === index })}
+                      className={styles.optionBasic}
                       style={{ height: virtualRow.size, transform: `translateY(${virtualRow.start}px)` }}
                     >
-                      <Stack direction="row" alignItems="center">
-                        <Checkbox
-                          key={id}
-                          value={allItemsSelected || isSelected}
-                          indeterminate={isAll && selectedItems.length > 0 && !allItemsSelected}
-                          aria-labelledby={id}
-                          onClick={(e) => {
-                            e.stopPropagation();
-                          }}
-                        />
-                        <OptionListItem
-                          label={
-                            isAll
-                              ? (item.label ?? item.value.toString()) +
-                                (isAll && inputValue !== '' ? ` (${options.length - 1})` : '')
-                              : (item.label ?? item.value.toString())
-                          }
-                          description={item?.description}
-                          id={id}
-                        />
+                      <Stack direction="column" justifyContent="space-between" width={'100%'} height={'100%'} gap={0}>
+                        {startingNewGroup && (
+                          <div className={styles.optionGroup}>
+                            <OptionListItem
```

Comment:
From a semantic pov it is a little bit weird to have this inside the list item. I don't know if we can/want to do something about it, as the `virtualRow.size` should be on one item.

Reply:
🤷‍♀️ I tried also to render to list items when there is a new group, one with the group name and the next one with the option info. The problem is that options move when I hover over them 🙃